src/components/App.js

```diff
@@ -25,12 +25,9 @@ export class App extends Component {
         <Route exact path='/login' component={LoginForm} />
         <Route
           exact
-          path='/movies/:id'
+          path='/movie/:id'
           render={({ match }) => {
-            const { id } = match.params;
-            let moviesData = this.props.allMovies.find(
-              movie => movie.id === parseInt(id)
-            );
+            let moviesData = [...this.props.movies];
             return <MovieShowPage {...moviesData} />;
```

Comment:
Can we double check this section?  It looks like a merge conflict might have accidentally removed some lines we need?